trading/trading.go

```diff
@@ -24,6 +24,10 @@ type Window struct {
 	Symbols stringset.StringSet
 }
 
+func (w Window) MeanPrice(symbol string) float64 {
```

Comment:
Yeah I have no idea what would be best.

What we're trying to compute usually is like "roughly how much are these stocks in this window." I would hope that in the aggregate, (High + Low)/2 ~= (Open + Close) / 2, but if we learn otherwise we can update this.

I'll name the function MeanHighLowPrice for now to be more explicit.